Dalamud.Boot/xivfixes.h

```diff
@@ -7,6 +7,7 @@ namespace xivfixes {
     void redirect_openprocess(bool bApply);
     void backup_userdata_save(bool bApply);
     void clr_failfast_hijack(bool bApply);
```

Comment:
If you're in here, you can probably delete this one, it was just an experiment and never worked correctly

Reply:
Should be gone now